runtime/opt/taupage/init.d/01-register-scalyr-agent.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+import logging
+import subprocess
+import re
+import boto.utils
+import json
+import ast
+
+
+from taupage import get_config
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+scalyr_agent_config_file = '/etc/scalyr-agent-2/agent.json'
+
+main_config = get_config()
+logging_config = main_config.get('logging')
+
+mount_custom_log = main_config.get('mount_custom_log')
+rsyslog_aws_metadata = main_config.get('rsyslog_aws_metadata')
+
+application_log_path = '/var/log/application.log'
+custom_log_path = '/var/log-custom/*.log'
+auth_log_path = '/var/log/auth.log'
+sys_log_path = '/var/log/syslog'
+
+jwt_redaction = [
+    {
+        'match_expression': 'eyJ[a-zA-Z0-9/+_=-]{5,}\\.eyJ[a-zA-Z0-9/+_=-]{5,}\\.[a-zA-Z0-9/+_=-]{5,}',
+        'replacement': '+++JWT_TOKEN_REDACTED+++'
+    }
+]
+
+if rsyslog_aws_metadata:
+    sys_log_parser = 'systemLogMetadata'
+else:
+    sys_log_parser = 'systemLog'
+
+if not logging_config:
```

Comment:
I'd default `logging_config` to `{}` and drop this whole section. Otherwise you have 3 cases for most of the configuration settings: no configuration present, configuration present but key undefined and configuration present and key defined. I don't understand why the first two need to be different.